variants/sea_picro/pins_arduino.h

```diff
@@ -0,0 +1,41 @@
+#pragma once
+
+// Pin definitions taken from:
+//    https://github.com/joshajohnson/sea-picro/blob/master/documentation/pinout/sea-picro-top-pinout-rory-render.png
+
+// LEDs not pinned out
+#define PIN_LED (31u)
+
+// Serial
+#define PIN_SERIAL1_TX (0u)
+#define PIN_SERIAL1_RX (1u)
+
+// Not pinned out
+#define PIN_SERIAL2_TX (31u)
+#define PIN_SERIAL2_RX (31u)
+
+// SPI
+#define PIN_SPI0_MISO (20u)
+#define PIN_SPI0_MOSI (23u)
+#define PIN_SPI0_SCK (22u)
+#define PIN_SPI0_SS (31u) // not pinned out
```

Comment:
But per [pinout diagram](https://github.com/joshajohnson/sea-picro/blob/master/documentation/pinout/sea-picro-top-pinout-rory-render.png), GPIO21 is nCS (=SS)? Would also match SPI0 CSn for GPIO21 per [datasheet](https://datasheets.raspberrypi.com/rp2040/rp2040-datasheet.pdf).